content/02.introduction.md

```diff
@@ -110,7 +110,8 @@ A later study reported radiographic findings such as ground-glass opacity and bi
 
 COVID-19 can affect diverse body systems in addition to causing respiratory problems.
 For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].
-It can even cause neurological complications [@doi:10.1016/j.bbi.2020.03.031], potentially including stroke, seizures or meningitis [@doi:10.1002/jmv.26000].
+It can also cause neurological complications [@doi:10.1016/j.bbi.2020.03.031], potentially including stroke, seizures or meningitis. [@doi:10.1002/jmv.26000][@doi:10.1002/ana.25807]
```

Comment:
```suggestion
It can also cause neurological complications [@doi:10.1016/j.bbi.2020.03.031], potentially including stroke, seizures or meningitis [@doi:10.1002/jmv.26000; @doi:10.1002/ana.25807].
```